appinventor/appengine/src/com/google/appinventor/client/style/classic/variableColors.css

```diff
@@ -0,0 +1,1248 @@
+/*
+* Copyright 2007-2011 Google Inc. All Rights Reserved.
+* Copyright 2011-2017 MIT, All Rights Reserved.
+* Released under the Apache License, Version 2.0
+* http://www.apache.org/licenses/LICENSE-2.0
+*/
```

Comment:
Please fix the preamble to be the same as the other files. Also, since this is new there shouldn't be copyright attribution to Google.